src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Completion/Delegation/DelegatedCompletionListProviderTest.cs

```diff
@@ -292,7 +292,7 @@ await _provider.GetCompletionListAsync(
 
     [Theory]
     [InlineData("$$", true)]
-    [InlineData("<$$", true)]
+    [InlineData("<$$", false)]
     [InlineData(">$$", true)]
```

Comment:
This is a behavior change from the legacy editor. Legacy editor would've included snippets here. If we are trying to reduce friction while removing legacy editor, this might be a somewhat important scenario. E.g. previously (in legacy) you could type in "<" to invoke html completion, scroll or type more to select snippet, and tab to complete snippet. With this change snippets will be absent (and possibly they should've been present in more cases). 

I want to make sure we realize there is a behavior change here that affects end users and are fine with it if we are to go through with this change.

Reply:
Legacy:

![image](https://github.com/user-attachments/assets/5a6a1f3a-4174-4ac0-badb-1560c4c73512)

Reply:
This PR is not introducing that behaviour change, it was introduced when we added snippets. We don't show snippets when bringing up completion inside a tag, legacy does. This PR doesn't change that, it just makes our experience the same whether typing at the end of a document or in the middle.

Reply:
In that case we probably should've had a test case like ```<$$ <div></div>``` to illustrate that. This was before my time as active contributor, I assume we discussed this behavior difference and are fine with it?

Reply:
I don't remember specifically but given LSP snippets and legacy snippets work in a fundamentally different way, I don't think it could have happened any other way. I think of it more like "show a subset of snippets as completion items", rather than "support snippets in Razor". If we were to show snippets inside elements in the LSP editor the user would end up with two `<`s at the start of their tag, as the snippet includes one.

Reply:
Yeah, there is special code in the legacy HTML to see if there is an "<" in front of snippet shortcut. I guess we can leave it as is unless there are user complaints (especially since we already shipped it like this and haven't heard any complaints AFAIK). @danroth27 and @leslierichardson95  FYI.